server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/BallotManifestImport.java

```diff
@@ -91,6 +91,13 @@ private void updateCountyDashboard(final Response the_response,
     if (cdb == null) {
       serverError(the_response, "could not locate county dashboard");
     } else {
+      // mark any previous ballot manifest import as NOT_IMPORTED
+      if (cdb.manifestFile() != null) {
+        cdb.manifestFile().setStatus(FileStatus.NOT_IMPORTED);
```

Comment:
So this changes the meaning of `NOT_IMPORTED` to be "not both imported and active", vs. "not ever imported", right? What's the motivation for this?

Reply:
The motivation is to accurately reflect the state of the system as it currently exists; it is not really useful to have the information that a file was imported in the past, and its data was subsequently removed from the database and replaced with that of a different file before the audit began - the only reason to reimport a file is to correct a mistake. Having the original file still be in the database, and the logs indicating what happened, is enough to document the mistake. It might be a better idea to make that enum value be `IN_USE` or `CURRENT_IMPORT` or some similar thing, but changing that at this point is problematic in other ways.